source/features/clear-pr-merge-commit-message.tsx

```diff
@@ -15,6 +16,18 @@ function init(): void | false {
 		deduplicatedAuthors.add('Co-authored-by: ' + author);
 	}
 
+	// Preserve closing issues numbers when a pr is merged into a non-default branch since GitHub doesn't close them #4531
+	const baseBranch = select('.base-ref a')!.title.split(':')[1];
+	if (baseBranch !== await getDefaultBranch()) {
+		for (const keyword of select.all('.comment-body .issue-keyword[aria-label^="This pull request closes"]')) {
+			// This pull request closes issue #51.
+			// This pull request closes pull request #52.
+			const closingKeyword = keyword.textContent!.trim(); // Keep the keyword as-is (closes, fixes, etc.)
+			const [issue] = /#\d*/.exec(keyword.getAttribute('aria-label')!)!;
+			deduplicatedAuthors.add(closingKeyword + ' ' + issue);
```

Comment:
Why is this adding issues to `deduplicatedAuthors`? It's no longer `deduplicatedAuthors` so it should be renamed to `preservedContent` perhaps